react/components/Checkbox/index.js

```diff
@@ -52,9 +53,12 @@ class Checkbox extends PureComponent {
           }
         )}>
         <div
-          className={classNames('vtex-checkbox__container relative w1 h1', {
-            mr3: label,
-          })}>
+          className={classNames(
+            `vtex-checkbox__container relative w1 h1 ${Styles.checkboxContainer}`,
```

Comment:
Hou about this instead of adding this class?
1. Change `items-center` to `items-start` on container
2. Add `w-100` class to `label` element

Reply:
Nice! Works like a charm